src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function(a) {
+    return function(b) {
+      counter++;
+
+      if (counter <= 3) {
+        return a + b;
+      } else if (counter % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return a + b;
```

Comment:
You can write it without else statement using &&. This will avoid repetition of return